rel/files/app.config

```diff
@@ -1,4 +1,34 @@
 [
+ {kernel, [
+  {logger_level, warning},
+  {logger, [
+    %% Console logger
+    {handler, default, logger_std_h, #{}},
+    %% Disk logger for errors
+    {handler, disk_log, logger_disk_log_h,
+       #{config => #{
+           file => "{{mongooseim_log_dir}}/ejabberd.log",
```

Comment:
One more thing, can we finally rename the file to `mongooseim.log` ? :)

Reply:
Absolutely! 😃 
There were just some tests depending on the name, but that's fixable anyway 😛